package.json

```diff
@@ -27,6 +27,7 @@
     "ttest": "node ./tests/playwright-test/stable-test-runner/node_modules/@playwright/test/cli test --config=tests/playwright-test/playwright.config.ts",
     "ct": "playwright test tests/components/test-all.spec.js --reporter=list",
     "test": "playwright test --config=tests/library/playwright.config.ts",
+    "local-test": "node packages/playwright-core/lib/cli/cli.js test --config=tests/library/playwright.config.ts",
```

Comment:
```suggestion
```

That's the same as just "test" npm will figure out the paths so playwright resolves to the playwright package in our monorepo.